auto_yaw: false
read_wrench: 0
compensate_d: false
starting_setpt: [0.0, 0.0, -20.0]

ctrller_type: 0 # mpc = 0; dompc = 1; pid = 2

Kp_list: [24000.0, 24000.0, 24000.0, 0.2]
Ki_list: [10.0, 10.0, 10.0, 0.0]
Kd_list: [0.0, 0.0, 0.0, 0.0]

